reference/tax_provider.yml

```
@@ -843,7 +843,7 @@ paths:
       description: |-
         Replace the persisted tax quote (identified by the given unique ID) with the provided quote request (represented by the **AdjustRequest**).
 
-        Relevant for partial refunds, full refunds, returns, and other Order modifications where there have been changes to the tax liabilities.
+        Relevant for returns, partial refunds, and other Order modifications where there have been changes to the tax liabilities.
```

```suggestion
        Relevant for returns, partial refunds, and other order modifications where there have been changes to the tax liabilities.
```

🍹 While we are on the line, I think we are trying to do better with this style of capitalisation.

---

We're working on the Style Guide, and we'll probably change all the capitalization at once so it's not still half capitalized in some places.